src/panels/developer-tools/service/developer-tools-service.js

```diff
@@ -83,6 +88,14 @@ class HaPanelDevService extends LocalizeMixin(PolymerElement) {
         .error {
           color: var(--error-color);
         }
+
+        :host([rtl]) .desc-container {
+          text-align: right;
+        }
+
+        :host([rtl]) .desc-container h3 {
+          direction: ltr;
```

Comment:
Should we make this auto so it will check the contents if it is rtl or ltr?
```suggestion
          direction: auto;
```

Reply:
No because these seem to be non localized server notifications which are always going to LTR and it looks great. Now atleast the dot or multi-sentence will be readable.